homework/12_A/24_Yavor_Pachedjiev/homework01/test_homework.py

```diff
@@ -0,0 +1,23 @@
+from homework import *
+
+
+def test_matching():
+    assert check_Circles((-7,5), 5, (-7,5), 5) == "Matching"
+
+def test_not_intersecting():
+    assert check_Circles((-7,5), 5, (13,15), 5) == "not intesecting"
+
+def test_touching():
+    assert check_Circles((5,5), 5, (15,5), 5) == "touching"
+
+def test_intersecting():
+    assert check_Circles((5,5), 5, (15,5), 7) == "intersecting"
+
+test_matching()
```

Comment:
No need to call these functions explicitly. Pytest will call them automatically.